### ISSUE TITLE
Point the ingestion server to the new sensitive tables

### ISSUE THREAD

**Issue by AetherUnbound** [WordPress/openverse#2628]
## Description

Update table references on the ingestion server from the previous mature-related tables to the new sensitivity-related ones. Instructions for how to perform this change are present in the implementation plan associated with this work: https://docs.openverse.org/projects/proposals/trust_and_safety/content_report_moderation/20230517-implementation_plan_mature_copy_change.html#code (steps 6).

A deployment of the ingestion server will be necessary before handling downstream tickets.

## Additional context

Part of #383, blocked by #2627

**Comment by AetherUnbound** [comment]
Closing in favor of #3723 based on #3568.
